tests/js/fixtures/pageFilters.ts

```diff
@@ -13,3 +15,29 @@ export function PageFiltersFixture(params: Partial<PageFilters> = {}): PageFilte
     ...params,
   };
 }
+
+export function GetPageFiltersStorageFixture(params = {}) {
+  return {
+    pinnedFilters: new Set<PinnedPageFilter>(['projects']),
+    state: {
+      project: [],
+      environment: [],
+      start: null,
+      end: null,
+      period: '14d',
+      utc: null,
+    },
+    ...params
+  };
+}
+
+export function PageFilterStateFixture(params: Partial<PageFiltersState> = {}) {
```

Comment:
```suggestion
export function PageFilterStateFixture(params: Partial<PageFiltersState> = {}): PageFiltersState {
```
this will ensure that future changes to filters fail the types we define in this fixture as opposed to the call sites failing